src/app/core/services/phase.service.ts

```diff
@@ -178,6 +183,7 @@ export class PhaseService {
           // No Session Fix Necessary
           return of(null);
         } if (sessionFixPermission === true) {
+          isSessionFixPermissionGranted = sessionFixPermission;
```

Comment:
Should we move this "caching" step into an independent step that precedes the repo creation step? (perhaps using [tap](https://rxjs-dev.firebaseapp.com/api/operators/tap) operator)

What do you think?

Reply:
Sure! I'll look into this